src/nix/devcontainer-feature.json

```diff
@@ -0,0 +1,26 @@
+{
+    "id": "nix",
+    "version": "1.0.0",
+    "name": "Nix",
+    "documentationURL": "https://github.com/devcontainers/features/tree/main/src/nix",
+    "description": "Installs Nix which is a tool that takes a unique approach to package management and system configuration",
+    "options": {
```

Comment:
Would it make sense to have an option for a set of Nix packages to include in the installation? That way the package downloads can be done during an image prebuild to save time later on when users create their dev environment from the image.